hledger/Hledger/Cli/Commands/Balance.hs

```diff
@@ -283,6 +283,7 @@ balancemode = (defCommandMode $ ["balance"] ++ aliases) { -- also accept but don
      ,flagReq  ["format"] (\s opts -> Right $ setopt "format" s opts) "FORMATSTR" "use this custom line format (in simple reports)"
      ,flagNone ["pretty-tables"] (\opts -> setboolopt "pretty-tables" opts) "use unicode when displaying tables"
      ,flagNone ["sort-amount","S"] (\opts -> setboolopt "sort-amount" opts) "sort by amount instead of account name"
+     ,flagNone ["forecast"] (\opts -> setboolopt "forecast" opts) "generate forecast transactions"
```

Comment:
You can add it to `reportflags` in `hledger/Hledger/Cli/CliOptions.hs` to get all report commands affected similar to how `--yearly` and `--costs` are defined and can be seen even in `stat` and `rewrite`.

We may want to introduce `budgetingFlags` or something like that if we want target subset of reports.

Reply:
Thanks, i missed reportflags. Fixing